packages/react/src/components/FlyoutMenu/FlyoutMenu.jsx

```diff
@@ -44,7 +44,7 @@ const DefaultFooter = ({ setIsOpen, onCancel, onApply, i18n }) => (
     <Button
       className={`${iotPrefix}--flyout-menu__cancel`}
       kind="secondary"
-      testID="flyout-menu-cancel"
+      testId="flyout-menu-cancel"
```

Comment:
are we making these all configurable for consistency? Looks like this one is all hard coded.

Reply:
Will be addressed in our v3 branch to avoid breaking changes

Reply:
Yes, it should be configurable, but that DefaultFooter component in there isn't exported and is built-in to the `FlyoutMenu`, so if I were to pass testId to it--inheriting the testId passed to the `FlyoutMenu`, then that testId changes and could break tests, so I was leaving that for v3. For example, the Advanced Filter Flyout uses it. and I pass `advanced-filter-flyout` as the testId to it. If I pass that testId along to that `DefaultFooter` then, those buttons become `advanced-filter-flyout-menu-cancel` or `advanced-filter-flyout-menu-save` and our own internal tests fail.